synfig-studio/src/gui/duckmatic.h

```diff
@@ -247,6 +251,9 @@ class Duckmatic
 
 	void connect_signals(const Duck::Handle &duck, const synfigapp::ValueDesc& value_desc, CanvasView &canvas_view);
 
+	double calculate_distance_from_guide(const Guide& guide, const synfig::Point& point);
+	double calculate_distance_from_guide(const Guide& guide, const synfig::Point& point)const { return const_cast<Duckmatic*>(this)->calculate_distance_from_guide(guide,point); }
+
```

Comment:
Looking into the implementation, there is no need to the non-const version.

Reply:
oh yes definitely. nice notice :).